python/lsst/dax/imgserv/api_soda.py

```diff
@@ -161,19 +175,33 @@ def imagesoda_unhandled_exceptions(error):
     return resp
 
 
+def _service_response(soda_method):
+    """ Return service info using DALI template.
+    """
+    service_ep = os.getenv("IMGSERV_FQDN", "localhost:5000")
+    prefix = "http://" if "localhost" in service_ep else "https://"
+    service_url = prefix + service_ep + "/api/image/soda/"+soda_method
+    resp = make_response(render_template("soda_descriptor.xml",
+                                         soda_ep=service_url),
+                         HTTPStatus.OK)
+    resp.headers["Content-Type"] = "text/xml"
+    return resp
+
+
 @image_soda.errorhandler(HTTPStatus.NOT_FOUND)
 def _image_not_found():
     """ Return a generic error using DALI template.
     """
     message = "Image Not Found"
-    resp = make_response(render_template("dali_error.xml",
-                                         dali_error_msg=message),
+    resp = make_response(render_template("dali_response.xml",
+                                         dali_resp_state="Error",
+                                         dali_resp_msg=message),
                          HTTPStatus.NOT_FOUND)
     resp.headers["Content-Type"] = "text/xml"
     return resp
 
 
-def _getparams():
+def _getparams(mark_api=True):
```

Comment:
What is mark_api?  This seems very confusing?  Why did you add this as an option?  I don't see anyone passing in false

Reply:
Removed.  Was useful earlier during coding.